File: API/src/main/java/com.healthtracker.tracker_api/model/LogInModel.java
package com.healthtracker.tracker_api.model;

import lombok.Getter;

@Getter
public class LogInModel {
    private final String username;
    private final String password;

    public LogInModel(String u, String p) {
        this.username = u;
        this.password = p;
    }
}

File: MVVM/Models/LogInModel.cs
namespace HealthTrackerMVVM.Models
{
    public class LogInModel
    {
        public string Username { get; set; }

        public string Password { get; set; }

        public LogInModel()
        {
            this.Username = string.Empty;
            this.Password = string.Empty;
        }
    }
}
